api/src/main/resources/application.properties

```diff
@@ -23,6 +23,12 @@ spring.jpa.show-sql=false
 spring.jpa.hibernate.ddl-auto=validate
 server.error.include-message=always
 
+## Keep the log levels under control
+spring.main.banner-mode=off
+spring.jpa.open-in-view=false
```

Comment:
This specific property does more than just trim log messages. I'm fine with setting it if it isn't breaking functionality, but I think it should live alongside the other JPA properties.

Reply:
shouldn't break anything if you're not using spring mvc views, which i don't think you are. I'll move it.